frontend/src/components/TrafficList/TrafficDetails.tsx

```diff
@@ -144,7 +144,7 @@ class TrafficDetails extends React.Component<TrafficDetailsProps, TrafficDetails
           </Grid>
         </RenderComponentScroll>
         <TimeDurationModal
-          customDuration={false}
+          customDuration={true}
```

Comment:
And similarly here, wasn't `customDuration={false}` right already?

Reply:
You are right. Forgot about this tab.